utils/scanUtils.py

```diff
@@ -453,7 +453,7 @@ def sbitRateScanAllLinks(args, rateTree, vfatBoard, chan=128, scanReg="CFG_THR_A
 
     # Make the containers
     nDACValues = (args.scanmax-args.scanmin+1)/args.stepSize
-    arraySize = amcBoard.nOHs * nDACValues
+    arraySize = 12 * nDACValues
     scanDataDAC = (c_uint32 * arraySize)()
     scanDataRate = (c_uint32 * arraySize)()
     scanDataRatePerVFAT = (c_uint32 * (24 * arraySize))() # per VFAT
```

Comment:
I suppose that in all upstream places, this `24` is also hard coded, so we don't have any issue there?

Reply:
That's a good question. It seems that the number of VFATs is *not* hardcoded, but passed as an argument or set a compile time. However, it is safe to use an array which is bigger than necessary.

I don't currently have a setup to test a patch where 24 is replaced with `nVFATs` and wehre `nVFATs` is actually different than 24. However, if the code did work previously with GE2/1, there is no reason it wouldn't work anymore.